checks/fuzzing.go

```diff
@@ -26,16 +27,33 @@ import (
 // CheckFuzzing is the registered name for Fuzzing.
 const CheckFuzzing = "Fuzzing"
 
+var (
+	ossFuzzRepo      clients.RepoClient
+	errOssFuzzRepoCh chan error
+	errOssFuzzRepo   error
+	okToRead         bool
+	once             sync.Once
+)
+
 //nolint:gochecknoinits
 func init() {
 	registerCheck(CheckFuzzing, Fuzzing)
 }
 
 // Fuzzing runs Fuzzing check.
 func Fuzzing(c *checker.CheckRequest) checker.CheckResult {
-	ossFuzzRepo := githubrepo.CreateGithubRepoClient(c.Ctx, c.Client, c.GraphClient)
-	if err := ossFuzzRepo.InitRepo("google", "oss-fuzz"); err != nil {
-		e := sce.Create(sce.ErrScorecardInternal, fmt.Sprintf("InitRepo: %v", err))
+	once.Do(func() {
+		ossFuzzRepo = githubrepo.CreateGithubRepoClient(c.Ctx, c.Client, c.GraphClient)
```

Comment:
I think we can just set ossFuzzRepo and errOssFuzzRepo directly without involving any channels.

Reply:
Done :)